gradle.properties

```diff
@@ -23,4 +23,4 @@
 
 org.gradle.daemon=true
 org.gradle.parallel=true
-org.gradle.jvmargs=-XX:MaxPermSize=1024m
+org.gradle.jvmargs=-Xmx5120M
```

Comment:
I'm not sure if this should be in the repository since a meaningful value depends on each developer's own hardware